src/backend/src/prisma/seed.ts

```diff
@@ -182,10 +214,18 @@ const performSeed: () => Promise<void> = async () => {
     ['Decrease weight by 90% from 4.8 pounds to 0.48 pounds'],
     ['Provides removable section for easy access to the pedal box'],
     ['Compatible with a side-pod chassis design'],
-    'https://youtu.be/dQw4w9WgXcQ',
-    'https://youtu.be/dQw4w9WgXcQ',
-    'https://youtu.be/dQw4w9WgXcQ',
-    'https://youtu.be/dQw4w9WgXcQ',
+    [
+      {
+        linkId: '-1',
```

Comment:
same thing

Reply:
This is just to symbolize a non created link